real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/LoggingPanel.java

```diff
@@ -100,6 +116,20 @@ public double getPlaybackSpeed() {
 			return 1;
 		}
 	}
+	
+	/**
+	 *  ResetEncoderButtonHandler
+	 * @author Sean
+	 * 
+	 * Sends a reset request message to low level when the "Reset Encoder" 
+	 * button on the gui is pressed
+	 *
+	 */
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Each class should declare at least one constructor](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2230250215/issues/source?bid=2938665&fileBranchId=3172754#l128)